src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/SharedArrayBuffer.cs

```diff
@@ -23,6 +23,22 @@ internal SharedArrayBuffer(IntPtr jsHandle, bool ownsHandle) : base(jsHandle, ow
         /// <value>The size, in bytes, of the array.</value>
         public int ByteLength => (int)GetObjectProperty("byteLength");
 
+        /// <summary>
+        /// Returns a new JavaScript Core SharedArrayBuffer whose contents are a copy of this SharedArrayBuffer's bytes from begin = 0,
+        /// inclusive, through to the end of the sequence, exclusive.
+        /// </summary>
```

Comment:
This method doesn't take a `begin`, or `end` args, so this description needs to be reworded.

Reply:
Addressed in recent commits